### PR TITLE
Fix  #30 : Contact Us API Integration

### LINKED ISSUE #30: Contact Us API Integration
This issue tracks the integration of the Contact Us API into our codebase. 
[Reference link](https://radiant-thicket-94737.herokuapp.com)

### PR DESCRIPTION
**Pull request by AK1406**
Fixes #30

Implementation of Contact Us APIs.

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+TechnoJam
```
Comment (Sarthak2601) [inline .idea/.name]: We don't need this file, please delete this.

File: .idea/compiler.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <bytecodeTargetLevel target="1.8" />
+  </component>
+</project>
```
Comment (Sarthak2601) [inline .idea/compiler.xml]: Ditto as above.

File: .idea/jarRepositories.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <option name="name" value="BintrayJCenter" />
+      <option name="url" value="https://jcenter.bintray.com/" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="Google" />
+      <option name="name" value="Google" />
+      <option name="url" value="https://dl.google.com/dl/android/maven2/" />
+    </remote-repository>
+  </component>
+</project>
```
Comment (Sarthak2601) [inline .idea/jarRepositories.xml]: Ditto. We don't need to push the .idea files as every system will have its own set of files.

File: .idea/misc.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
```
Comment (Sarthak2601) [inline .idea/misc.xml]: Please delete.

File: app/src/main/java/com/sarthak/technojam/data/contactUs/ContactPost.kt [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+    @SerializedName("name")
+    val name: String,
+    @SerializedName("email")
+    val email: String,
+    @SerializedName("phnNo")
+    val phnNo : Long,
+    @SerializedName("query")
+    val query : String
+)
```
Comment (Sarthak2601) [inline app/src/main/java/com/sarthak/technojam/data/contactUs/ContactPost.kt]: Please add EOF. (End of file)

File: gradle/wrapper/gradle-wrapper.properties [hunk 6]
```diff
@@ -1,6 +1,6 @@
-#Thu Oct 01 05:09:23 IST 2020
+#Thu Dec 17 11:27:39 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (Sarthak2601) [inline gradle/wrapper/gradle-wrapper.properties]: I don't think these changes are required. Any specific reason of pushing them?

**Review by Sarthak2601 — CHANGES_REQUESTED** [review]
The overall implementation looks fine but there are some unnecessary files that must be removed and some nits to take care of.

File: gradle/wrapper/gradle-wrapper.properties (hunk 6 above)
Comment (AK1406) [reply gradle/wrapper/gradle-wrapper.properties]: not any reason ! I must had cared about all these and unchecked unwanted files while pusing . 
Done : deleted .idea files , added EOF in ContactPost

File: gradle/wrapper/gradle-wrapper.properties [hunk 7]
```diff
@@ -1,4 +1,4 @@
-#Thu Oct 01 05:09:23 IST 2020
+#Thu Dec 17 11:27:39 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
```
Comment (Sarthak2601) [inline gradle/wrapper/gradle-wrapper.properties]: Please delete this file as well.

Comment (AK1406) [reply gradle/wrapper/gradle-wrapper.properties]: Done

**Review by Sarthak2601 — APPROVED** [review]
Nice work @Anju1415!

**Review by Areeb786123 — APPROVED** [review]

**Pull request merged**